cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -243,6 +243,25 @@ private static void writeSynchronousMachines(Network network, String cimNamespac
 
             addRegulatingControlView(g, regulatingControlViews, context);
         }
+        for (Battery b : network.getBatteries()) {
+            CgmesExportUtil.writeStartAbout("SynchronousMachine", context.getNamingStrategy().getCgmesId(b), cimNamespace, writer, context);
+            writer.writeStartElement(cimNamespace, "RegulatingCondEq.controlEnabled");
+            writer.writeCharacters("false"); // TODO handle battery regulation
+            writer.writeEndElement();
+            writer.writeStartElement(cimNamespace, "RotatingMachine.p");
+            writer.writeCharacters(CgmesExportUtil.format(-b.getTargetP()));
+            writer.writeEndElement();
+            writer.writeStartElement(cimNamespace, "RotatingMachine.q");
+            writer.writeCharacters(CgmesExportUtil.format(-b.getTargetQ()));
+            writer.writeEndElement();
+            writer.writeStartElement(cimNamespace, "SynchronousMachine.referencePriority");
+            // reference priority is used for angle reference selection (slack)
+            writer.writeCharacters(isInSlackBus(b) ? "1" : "0");
+            writer.writeEndElement();
+            writer.writeEmptyElement(cimNamespace, "SynchronousMachine.operatingMode");
+            writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, cimNamespace + "SynchronousMachineOperatingMode.generator"); // FIXME when is a battery motor and when is it generator?
```

Comment:
A way to express that the battery may produce or consume active power is set its mode to `generatorOrMotor`. Then in the SSH it can be specified which is the current mode: `SynchronousMachine.operatingMode`, that can be `generator` or `motor`.